src/Microsoft.SymbolStore.Client/SymbolLocator.cs

```diff
@@ -0,0 +1,162 @@
+﻿using FileFormats.PDB;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FileFormats;
+
+namespace Microsoft.SymbolStore.Client
+{
+    public sealed class SymbolLocator
+    {
+        private static string[] s_microsoftSymbolServerUrls = new string[] { "http://msdl.microsoft.com/download/symbols", "https://nuget.smbsrc.net", "http://referencesource.microsoft.com/symbols", "https://dotnet.myget.org/F/dotnet-core/symbols" };
```

Comment:
This line should be broken up (it scrolls right off the side of my fairly high resolution monitor)